notebooks/OpenMLxProbabl-hackathon.ipynb

```diff
@@ -0,0 +1,214 @@
+    "\n",
+    "- **Curated issues** : Most of our external contributors have enough on their plate already. Having a curated list of issues before sprints would ensure that no time is wasted and let our contributors focus on the tasks suitable for them.\n",
+    "\n",
+    "- **Different tiers of events** : To ensure that everyone is given tasks they can handle, it would be nice to have separate events for contributors with varying levels of expertise. This would also have the added benefit of retaining beginner-friendly issues for sprints to prevent more experienced contributors from claiming them early.\n",
+    "\n",
+    "- **Mentoring** : Incorporating one-to-one mentoring to help new contributors set up their development environments would help them feel more connected to the project. It is understandable that this is time consuming, so perhaps some way of deciding who gets mentored can be set up in time.\n",
+    "\n",
+    "- **Contributors guide** : Simplifying the contributor's guide and adding video tutorials would make it a lot more beginner-friendly, especially to those who have never filed a PR before.\n",
+    "\n",
+    "- **Semi regular events** : Some of our colleagues found that they tended to care about a project more if there were semi regular events they could set time aside for. Having these helped them slowly build a sense of community as well.\n",
+    "\n",
+    "- **Incentives** : A common question that many developers have is why they should even bother contributing. Helping them understand how contributing to open source projects can aid their careers, bring them closer to the community and also help them get internships would be a good start.\n",
+    "\n",
+    "<div style=\"text-align:center\">\n",
+    "<img alt=\"Discussions\" src=\"../images/blogs/openmlxsci/discussion.webp\" style=\"width:40%\">\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Governance, Funding, and Sponsorship\n",
+    "\n",
+    "This focus of this discussion was the governance structures of open-source projects, sustainable funding models, and the role of sponsorships in supporting project activities.\n",
+    "\n",
+    "**Takeaways:**\n",
+    "\n",
+    "- **Evolving Governance** : Since governance is not static, we can treat it as a living document that evolves with the needs of the community.\n",
+    "\n",
+    "- **Communication**: It is good practise to maintain open communication channels, such as mailing lists and monthly meetings. This keeps all interested contributors in the loop.\n",
+    "\n",
+    "- **Reviews**: A major challenge is that of approving new reviews in a large organization without passing through too many hoops. How to manage this is still an open question.\n",
```

Comment:
Reviews? What do you mean?
Is it supposed to reviewers, or pull requests, perhaps? Not sure.